Source/Moq.Analyzers/Moq.Analyzers.csproj

```diff
@@ -19,10 +19,14 @@
     <PackageReleaseNotes>A changelog is available at https://github.com/Litee/moq.analyzers/releases</PackageReleaseNotes>
     <Copyright>2017 Andrey Lipatkin</Copyright>
     <PackageTags>moq, tdd, mocking, mocks, unittesting, agile, unittest, mock, test, analyzers</PackageTags>
-    <DevelopmentDependency>true</DevelopmentDependency> 
+    <DevelopmentDependency>true</DevelopmentDependency>
   </PropertyGroup>
 
   <ItemGroup>
+    <PackageReference Include="Meziantou.Analyzer">
```

Comment:
We should lift this and the `Roslynator.Analyzers` out of this .csproj and into `//build/targets/compiler/Compiler.props` (or create another vertical slice for code analysis, your call).

Reply:
I moved it and fixed issues that were in the test code. I can't properly fix the issue in test until xUnit v3. See [xunit/#345](https://github.com/xunit/xunit/issues/345)